### PR TITLE
[MRG] Add pipelines documentation

### PR DESCRIPTION
**Pull request by annapasca**

### CODE REVIEW

**Comment by jasmainak** [comment]
@annapasca let us know when you are ready

File: doc/power.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+The power pipeline in the **sensor space** is implemented by the function 
+:py:func:`create_pipeline_power <ephypype.pipelines.power.create_pipeline_power>`
+and its Node :py:class:`power <ephypype.interfaces.mne.power.Power>`
+wraps the MNE Python functions  |welch|, |multitaper| computing the PSD
+using Welch's method and multitapers respectively.
+
+The power pipeline in the **source space** is implemented by the function 
+:py:func:`create_pipeline_power_src_space <ephypype.pipelines.power.create_pipeline_power_src_space>`
+and its Node :py:class:`power <ephypype.interfaces.mne.power.Power>` compute the PSD by the welch function
+of the scipy package.
```
Comment (jasmainak) [inline doc/power.rst]: why don't you move this info to the example we already? That way people know how to use this in code ...

File: doc/preproc_meeg.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+   <a href="http://martinos.org/mne/stable/auto_tutorials/plot_artifacts_correction_ica.html" target="_blank">here</a>
+
+In particular, the node:
+   
+* :py:class:`ds2fif <ephypype.nodes.import_data.ConvertDs2Fif>` converts from ds to fif data format 
+* :py:class:`preproc <ephypype.interfaces.mne.preproc.PreprocFif>` performs filtering on the raw data
+* :py:class:`ica <ephypype.interfaces.mne.preproc.CompIca>` computes ICA solution on raw fif data
+
+
+.. figure::  ./img/graph_dot_preproc.jpg
```
Comment (jasmainak) [inline doc/preproc_meeg.rst]: let's avoid adding `png` files if we can. It will increase the size of the repo.

**Comment by jasmainak** [comment]
@annapasca you still have images in your pull request. Are they really necessary?

**Comment by annapasca** [comment]
> @annapasca you still have images in your pull request. Are they really necessary?

I removed all except one!

**Comment by annapasca** [comment]
@jasmainak what do u think we miss before merge? Thks!

File: doc/source_reconstruction.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+In particular:
+   
+* :ref:`LF_computation`: compute the Lead Field matrix
+* :ref:`noise_covariance`: compute the noise covariance matrix
+* :ref:`inv_solution`: estimate the time series of the neural sources on a set of dipoles grid
+
+.. _inv_pipeline:
+
+.. figure::  ./img/graph_dot_inv.jpg
```
Comment (jasmainak) [inline doc/source_reconstruction.rst]: this does not render correctly: https://128-92522975-gh.circle-artifacts.com/0/html/source_reconstruction.html#source-reconstruction

File: doc/spectral_connectivity.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+:py:class:`plot_spectral <ephypype.interfaces.mne.spectral.PlotSpectralConn>`
+wraps the MNE Python functions  |spectral_connectivity| and |plot_connectivity_circle| to visualize the connectivity matrix as a circular graph.
+
+.. |spectral_connectivity| raw:: html
+
+   <a href="http://martinos.org/mne/stable/generated/mne.connectivity.spectral_connectivity.html?highlight=spectral_connectivity#mne.connectivity.spectral_connectivity" target="_blank">spectral_connectivity</a>
+
+
+   
+.. figure::  ./img/graph_dot_conn.jpg
```
Comment (jasmainak) [inline doc/spectral_connectivity.rst]: same here: https://128-92522975-gh.circle-artifacts.com/0/html/spectral_connectivity.html#spectral-connectivity

doesn't render correctly

File: doc/source_reconstruction.rst (hunk 3 above)
Comment (jasmainak) [reply doc/source_reconstruction.rst]: you should use the image from `auto_examples/...`

**Comment by jasmainak** [comment]
250 KB is too large for the image.

Try using something like tinypng.com to make it smaller. It should be at most 10-20 kB

File: doc/source_reconstruction.rst (hunk 3 above)
Comment (annapasca) [reply doc/source_reconstruction.rst]: in #64 there is no more this .rst

File: doc/spectral_connectivity.rst (hunk 4 above)
Comment (annapasca) [reply doc/spectral_connectivity.rst]: I pul the description in [plot_connectivity](https://github.com/neuropycon/ephypype/pull/54/commits/e11a5bf3a5ddf13ca00d27c8fe821a2ddfd27a24#diff-72cad35193d8d103241c702b9717fc4e)

**Comment by annapasca** [comment]
> 250 KB is too large for the image.
> 
> Try using something like tinypng.com to make it smaller. It should be at most 10-20 kB

I reduced to 60kB. Could it be ok?

**Comment by jasmainak** [comment]
Merged by squash (to avoid the .png bloating the repo). Thanks @annapasca !

**Comment by annapasca** [comment]
> Merged by squash (to avoid the .png bloating the repo). Thanks @annapasca !

Thanks for your revision!

**Pull request merged**